bench/app.py

```diff
@@ -100,7 +100,9 @@ def remove_from_excluded_apps_txt(app, bench_path='.'):
 
 def get_app(git_url, branch=None, bench_path='.', skip_assets=False, verbose=False, postprocess=True, overwrite=False):
 	if not os.path.exists(git_url):
-		if not check_url(git_url, raise_err=False):
+		if git_url.startswith('git@'):
+			pass
+		elif not check_url(git_url, raise_err=False):
```

Comment:
```suggestion
		if not git_url.startswith('git@') and not check_url(git_url, raise_err=False):
```

Reply:
Would it be better to handle it in the `check_url` function, which could just be calling new `check_ssh_url` and `check_https_url` functions?

Reply:
sure, that works too